flixel/addons/transition/TransitionTiles.hx

```diff
@@ -1,22 +1,25 @@
  package flixel.addons.transition;
+ 
 import flash.display.BitmapData;
+import flixel.addons.transition.TransitionEffect;
 import flixel.addons.transition.FlxTransitionSprite.TransitionStatus;
 import flixel.group.FlxGroup.FlxTypedGroup;
+import flixel.group.FlxSpriteGroup.FlxTypedSpriteGroup;
 
 /**
  * 
  * @author larsiusprime
  */
-class TransitionTiles extends Transition
+class TransitionTiles extends TransitionEffect
 {
-	private var _grpSprites:FlxTypedGroup<FlxTransitionSprite>;
+	private var _grpSprites:FlxTypedSpriteGroup<FlxTransitionSprite>;
```

Comment:
Are you asking why TransitionEffect is a FlxSpriteGroup or why TransitionTiles _grpSprites member is?

With TransitionEffect it's a FlxSpriteGroup because TransitionEffects can now be used apart from state transitions; for instance in Defender's Quest I use a TransitionTiles object to drive a healing spell visual effect. It helps to have an object that behaves like a FlxSprite so that you an easily move the x/y position of the whole thing around.

As for _grpSprites itself I suppose on further reflection there's no reason it *needs* to be a FlxTypedSpriteGroup as its just an internal type. Do you have a preference in this case?